### PR TITLE
how-to build jbooks

### PR DESCRIPTION
**Pull request by isabelizimm**
adding documentation for how to build a jupyter book

### CODE REVIEW

File: docs/build-jupyterbooks.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Publish on GitHub pages
+
+1. Run `pip install ghp-import` in cli interface
+2. Delete the `_build/html/_sources` as these are just copies of the repo files that were converted to html. (optional)
+3. Add `_build` folder to `.gitignore` so that its files are not tracked while pushing to master. (optional)
+4. Run `ghp-import -n -p -f _build/html`
+5. Build site by going to repo settings-->pages and set source to branch:gh-pages and directory:root/
+
+Within a few minutes, you should see your GitHub pages enivornment populate with your JupyterBook content.
```
Comment (pacospace) [inline docs/build-jupyterbooks.md:20]: `environment`

File: docs/build-jupyterbooks.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+At this point, you have created your Jupyter Book! The next steps are to publish your book as a GitHub pages site on GitHub.
+
+## Publish on GitHub pages
+
+1. Run `pip install ghp-import` in cli interface
+2. Delete the `_build/html/_sources` as these are just copies of the repo files that were converted to html. (optional)
+3. Add `_build` folder to `.gitignore` so that its files are not tracked while pushing to master. (optional)
+4. Run `ghp-import -n -p -f _build/html`
+5. Build site by going to repo settings-->pages and set source to branch:gh-pages and directory:root/
```
Comment (pacospace) [inline docs/build-jupyterbooks.md:18]: Some screenshots maybe? wdyt?

Comment (isabelizimm) [reply docs/build-jupyterbooks.md:18]: added a screenshot for step 5!

**Review by aakankshaduggal — COMMENTED** [review]
Thanks for putting this together @isabelizimm :smile: 
/lgtm

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**